patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -262,7 +261,24 @@
           }
        } else {
           return null;
-@@ -2320,9 +2357,10 @@
+@@ -2123,8 +2160,15 @@
+       this.field_70128_L = true;
+    }
+ 
+-   @Nullable
++   @Nullable @Deprecated // Use ITeleporter safe version
```

Comment:
If you put the Deprecated and comment on the next line or the previous line then this patch will just be an addition of one line instead of an addition and removal